app/models/manageiq/providers/microsoft/infra_manager/ps_scripts/get_inventory.ps1

```diff
@@ -85,10 +85,27 @@ $r["datastores"] = $diskvols
 $c = Get-SCVMHostCluster -VMMServer "localhost"
 $r["clusters"] = get_clusters($c)
 
-$e = Get-VMMServer -ComputerName "localhost"
+$e = Get-SCVMMServer -ComputerName "localhost"
 $r["ems"] = $e
 
-$file = [System.IO.Path]::GetTempFileName()
-$r | Export-CLIXML -Path $file -Encoding UTF8
-Get-Content $file -ReadCount 0
-Remove-Item -Force $file
+$inFile = [System.IO.Path]::GetTempFileName()
+$outFile = $inFile + '.gz'
+
+$r | Export-CLIXML -Path $inFile -Encoding UTF8
+
+$in = New-Object System.IO.FileStream $inFile, ([IO.FileMode]::Open), ([IO.FileAccess]::Read), ([IO.FileShare]::Read)
+$buf = New-Object byte[]($in.Length)
+$x = $in.Read($buf, 0, $in.Length)
+$in.Dispose()
+
+$out = New-Object System.IO.FileStream $outFile, ([IO.FileMode]::Create), ([IO.FileAccess]::Write), ([IO.FileShare]::None)
+
+$gStream = New-Object System.IO.Compression.GzipStream $out, ([IO.Compression.CompressionMode]::Compress)

```

Comment:
The VMM server itself, no. Research indicates that Windows will need the .NET framework 4.0 or later, though, I think. I can add this line to make it more robust:

Add-Type -assembly System.IO.Compression.FileSystem

Wasn't necessary against out QE box running Win2k12 with Powershell 4, though, nor did my Windows 8 VM require it.